Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/user-guide/how-to/run-dvc-on-windows.md

## POSIX-like command line experience

The regular Command Prompt (`cmd`) in Windows will most likely not help you use

> DVC effectively, or follow the examples in our docs. Make sure to avoid it. There's no
> perfect solution, but here are some ideas:

Reviewer: ```suggestion
DVC effectively, nor help you follow the examples in our docs. Here are some
alternatives:
```

By default, DVC tries to use [Less](<https://en.wikipedia.org/wiki/Less_(Unix)>)
as pager for the output of `dvc dag`. Windows doesn't have the `less` command
available however. Fortunately, there is a easy way of installing it via

> [Chocolatey](https://chocolatey.org/) (don't forget to install the tool first):

Reviewer: ```suggestion
[Chocolatey](https://chocolatey.org). After installing Chocolatey, run:
```
